Sources/ArcGISToolkit/Components/Authentication/TrustHostViewModifier.swift

```diff
@@ -60,42 +55,28 @@ struct TrustHostViewModifier: ViewModifier {
                 // it doesn't show.
                 isPresented = true
             }
-            .sheet(isPresented: $isPresented) {
-                VStack(alignment: .center) {
-                    title
-                        .padding(.vertical)
-                    message
-                        .padding(.bottom)
-                    HStack {
-                        Spacer()
-                        Button(role: .cancel) {
-                            isPresented = false
-                            challenge.resume(with: .cancel)
-                        } label: {
-                            Text(String.cancel)
-                                .padding(.horizontal)
-                        }
-                        .buttonStyle(.bordered)
-                        Spacer()
-                        Button(role: .destructive) {
-                            isPresented = false
-                            challenge.resume(with: .continueWithCredential(.serverTrust))
-                        } label: {
-                            Text(
-                                "Allow",
-                                bundle: .toolkitModule,
-                                comment: "A button indicating the user accepts a potentially dangerous action."
-                            )
-                            .padding(.horizontal)
-                        }
-                        .buttonStyle(.borderedProminent)
-                        Spacer()
+            .alert(
```

Comment:
I was wondering why this trust host prompt looked a bit off. It is because it makes much more sense as an alert rather than a sheet. I saw an Apple alert on my phone that looked just like this. Here is what this looks like now:
![Simulator Screenshot - iPhone 16 Pro Max - 2024-09-19 at 09 48 23](https://github.com/user-attachments/assets/5c97f9cf-744e-4dc9-b97a-b70bd6a75103)

Reply:
I think it is related to https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/381 where multiple alerts being presented at the same time will cause the following alerts to disappear.